controller/users_blackbox_test.go

```diff
@@ -51,6 +54,48 @@ func (s *TestUsersSuite) SecuredServiceAccountController(identity account.Identi
 	return svc, NewUsersController(svc, s.GormDB, s.Configuration)
 }
 
+func (s *TestUsersSuite) TestDeleteUsersBadRequest() {
+	// given
+	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Users(1), tf.Identities(1))
```

Comment:
```suggestion
	fxt := tf.NewTestFixture(s.T(), s.DB, tf.Identities(1))
```

Reply:
fixed in 27e29eca3292166ea064012056b9653c1a4b509b